ArchiSteamFarm/BotConfig.cs

```diff
@@ -184,6 +187,9 @@ public sealed class BotConfig {
 		[JsonProperty(Required = Required.DisallowNull)]
 		public readonly bool SendOnFarmingFinished = DefaultSendOnFarmingFinished;
 
+		[JsonProperty(Required = Required.DisallowNull)]
+		public readonly ImmutableHashSet<Steam.Asset.EType> CompleteTypesToSend = DefaultCompleteTypesToSend;
```

Comment:
You've added a new property but didn't specify serialization for it (below).

Reply:
Shall we disallow other values than `TradingCard` and `FoilTradingCard` in `CheckValidation()` for now? You previously said 

> In the future perhaps we can extend it to emoticons and backgrounds as well.

Is the future now already?

Reply:
It's a good idea, yes, this will ensure that users won't put other types there expecting them to work, and we can always remove this check when we allow them.

> Is the future now already?

Do you want to make this PR even bigger and hold it for even longer? 😉

Reply:
No. This is exactly why I wanted to add the check :D